flink-runtime/src/test/java/org/apache/flink/runtime/io/network/NettyShuffleEnvironmentBuilder.java

```diff
@@ -55,7 +54,9 @@
 
 	private MetricGroup metricGroup = UnregisteredMetricGroups.createUnregisteredTaskManagerMetricGroup();
 
-	private IOManager ioManager = new IOManagerAsync();
+	private final String tempDir = EnvironmentInformation.getTemporaryFileDirectory();
+
+	private String[] tempDirs = new String[] {tempDir};
```

Comment:
The `tempDirs` could be reset in builder, so it could not be defined final here. Do you think it is necessary for define static for both `tempDir` and `tempDirs`?

Reply:
I meant:
```
private static final String[] DEFAULT_TEMP_DIRS = new String[] {EnvironmentInformation.getTemporaryFileDirectory()};

private String[] tempDirs = DEFAULT_TEMP_DIRS;
```